code/modules/antagonists/blood_contract/blood_contract.dm

```diff
@@ -30,4 +30,4 @@
 		if(P == H || HAS_TRAIT(P, TRAIT_NO_MIDROUND_ANTAG))
 			continue
 		to_chat(P, "<span class='userdanger'>You have an overwhelming desire to kill [H]. [H.p_theyve(TRUE)] been marked red! Whoever [H.p_they()] [H.p_were()], friend or foe, go kill [H.p_them()]!</span>")
-		P.put_in_hands(new /obj/item/kitchen/knife/butcher(P), TRUE)
+		P.put_in_hands(new /obj/item/kitchen/knife/butcher,(P), TRUE)
```

Comment:
```suggestion
		P.put_in_hands(new /obj/item/kitchen/knife/butcher(P), TRUE)
```
`new /obj/item/kitchen/knife/butcher(P)` is a single command
it's a proc call to create a new instance of this type and put it in the hands of `P`
The `(P)` means to spawn it inside `P` (for some reason)